pyfr/plugins/tavg.py

```diff
@@ -104,7 +110,7 @@ def _prepare_exprs(self):
                 self.outfields.append(f'std-{k[4:]}')
 
             for k in cfg.items(cfgsect, prefix='fun-avg-'):
-                self.outfields.append(f'fun-std-{k[4:]}')
+                self.outfields.append(f'std-fun-{k[4:]}')
```

Comment:
Revert.

Reply:
This was delibrate, it makes filtering the fields much easier (and it makes more sense). std of the function rather than function of the std.

Reply:
Makes sense.